addon/components/sl-split-grid.js

```diff
@@ -0,0 +1,564 @@
+import Ember from 'ember';
+
+/**
+ * @module components
+ * @class  sl-split-grid
+ */
+export default Ember.Component.extend({
+
+    // -------------------------------------------------------------------------
+    // Dependencies
+
+    // -------------------------------------------------------------------------
+    // Attributes
+
+    /**
+     * HTML tag name of the root element
+     *
+     * @property {string} tagName
```

Comment:
All of the other files in this repository use Ember.String.  Please change the occurrence for this property as well as for all other similar instances in this file to be consistent with the rest of the codebase.  If it is desired to move away from the "Ember." syntax create an issue to be worked on separately.